app/routes/overview/components/utils.tsx

```diff
@@ -9,16 +9,18 @@ import styles from './Overview.css';
 
 export const renderMeta = (item: Event | Article) => {
   const isEvent = item.eventType ? true : false;
+
+  let format =
+    moment().year() === moment(item.startTime).year()
+      ? 'DD. MMM'
+      : 'DD. MMM YYYY';
+  if (isEvent) {
+    format += ' HH:mm';
+  }
```

Comment:
The exact time the article was posted isn't that important. However, it's different for events because it's important to know when it starts.